.vscode/settings.json

```diff
@@ -3,4 +3,23 @@
     "**/.git": true,
     "**/coverage": true
   },
+  "editor.quickSuggestions": {
```

Comment:
**question** (non-blocking): I'm not sure what these "quick suggestion" really have to do with the storybook upgrade? [The others for linting seem fine - although to be honest I'm not sure why we have vscode-specific settings saved as part of the repo, and it wasn't until discovering this in review that I realised we did!]

Reply:
It's quite common practice to have a `.vscode/settings.json` file to set up the basics of a vscode environment for onboarding purposes. You can then override these in your own vscode if there's issues with them. I added these as part of the mdx linting recommendations I believe but if there's a specific issue with any of them them I'm open to removing